lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_bump_version_hotfix.rb

```diff
@@ -10,17 +10,17 @@ def self.run(params)
         app = params[:app]
 
         current_version = Fastlane::Helper::Android::VersionHelper.get_release_version(product_name: app)
-        current_version_alpha = Fastlane::Helper::Android::VersionHelper.get_alpha_version(app)
-        new_version = Fastlane::Helper::Android::VersionHelper.calc_next_hotfix_version(params[:version_name], params[:version_code])
-        new_short_version = new_version_name
```

Comment:
This one particular line crashed because `new_version_name` variable doesn't exist. `new_short_version` wasn't even used in the rest of the method except for constructing the branch name below …